assimilation_code/modules/assimilation/adaptive_inflate_mod.f90

```diff
@@ -825,9 +807,9 @@ end subroutine bayes_cov_inflate
 !> Used to update density by taking approximate gaussian product
 !> original routine.
 
-function compute_new_density(dist_2, sigma_p_2, sigma_o_2, lambda_mean, lambda_sd, gamma, lambda)
+function compute_new_density(dist_2, sigma_p_2, sigma_o_2, lambda_mean, lambda_sd, gamma_corr, lambda)
```

Comment:
'gamma' is actually a Fortran intrinsic ... which is not the context here. To avoid confusion, I am trying to consistently use 'gamma_corr'  ... (which is what it was in other places, ... is this still a good name here?)